README.md

```diff
@@ -199,13 +199,18 @@ You can create a new reporter object that implements `{ log(logObject): () => {
 **Example:** Simple JSON reporter
 
 ```ts
-import { createConsola } from "consola";
+import { createConsola, LogTypes } from "consola";
 
 const consola = createConsola({
   reporters: [
```

Comment:
Using `reporters` as an option, disables all other reporters and makes console silent.

For this example, we can use `consola.addReporter` that adds an additional interceptor